cmd/soroban-cli/src/commands/contract/info/wasm_hash.rs

```diff
@@ -0,0 +1,67 @@
+use clap::{arg, command, Command, ArgMatches};
+use soroban_rpc::client::Client;
+use soroban_xdr::{ScAddress, ScVal, DecodeError};
+use stellar_xdr::XdrCodec;
+use crate::config::Config;
+use crate::error::Result;
+use clap::Parser;
+use stellar_xdr::curr as xdr;
+
+use crate::commands::{config::network, global};
+use crate::config::locator;
+use crate::rpc;
+
+#[derive(Parser, Debug, Clone)]
+#[group(skip)]
+pub struct Cmd {
+    #[command(flatten)]
+    pub config_locator: locator::Args,
+
+    #[command(flatten)]
+    pub network: network::Args,
+
+    /// Contract ID to get the wasm hash for
+    #[arg(long)]
+    pub contract_id: stellar_strkey::Contract,
```

Comment:
This should be a string, that is then used with alias resolution. Worth looking at other commands that accept a contract ID to see how this works.